### PR TITLE
refactor(*): use custom beamcoder

### PR DESCRIPTION
**Pull request by stepancar**
Sorry @antoineMoPa !!!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "test-in-docker": "docker build -t framefusion-test-image -f ./docker-images/ci.Dockerfile . && docker run -it framefusion-test-image"
   },
   "dependencies": {
-    "@antoinemopa/beamcoder": "^0.7.4",
```
Comment (animanathome) [inline package.json:22]: bye bye @antoineMoPa

**Review by animanathome — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (animanathome) [reply package.json:22]: welcome lumen5

Comment (antoineMoPa) [reply package.json:22]: @animanathome @stepancar 

image

**Comment by michaeljherrmann** [comment]
✌️

**Pull request merged**